scripts/docker-run-rootfs.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -e
+
+ROOTFS_PATH="$(pwd)/bin/targets/x86/64/openwrt-x86-64-generic-rootfs.tar.gz"
```

Comment:
I'm `scripts/qemustart` user, so I would welcome if this new script could be similar.

It would be nice if one could specify custom rootfs via `--rootfs <rootfs>` (I usually just wget some latest image)

Reply:
It would probably be also nice to allow using of some prebuilt Docker images as well.

Reply:
added via --rootfs and --prebuild